test/e2e/framework/node/resource.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	netutil "k8s.io/utils/net"
```

Comment:
I think most other import aliases for this package has been `utilnet`?

Reply:
I have seen it a few different ways, but I'm happy to switch to this format since the majority seems to favor it.

```
cmluciano@starlord:~/go/src/k8s.io/kubernetes$ ack 'netutil "k8s.io/utils/net"'|wc -l
6
cmluciano@starlord:~/go/src/k8s.io/kubernetes$ ack 'netutils "k8s.io/utils/net"'|wc -l
14
cmluciano@starlord:~/go/src/k8s.io/kubernetes$ ack 'utilnet "k8s.io/utils/net"'|wc -l
42

```

Reply:
hmm okay, feel free to ignore then